go.mod

```diff
@@ -9,10 +9,10 @@ require (
 	github.com/cespare/xxhash/v2 v2.1.1
 	github.com/client9/misspell v0.3.5-0.20180309020325-c0b55c823952 // indirect
 	github.com/dlclark/regexp2 v1.2.0 // indirect
-	github.com/dop251/goja v0.0.0-20200414142002-77e84ffb8c65 // indirect
+	github.com/dop251/goja v0.0.0-20200526165454-f1752421c432 // indirect
 	github.com/dop251/goja_nodejs v0.0.0-20200128125109-2d688c7e0ac4 // indirect
 	github.com/dustin/go-humanize v0.0.0-20171111073723-bb3d318650d4
-	github.com/elastic/beats/v7 v7.0.0-alpha2.0.20200513130238-8992fa169aaa
+	github.com/elastic/beats/v7 v7.7.1-0.20200603124025-36c4ff678d3c
```

Comment:
not quite certain why version now changes from `7.0.0-alpha2` to  `7.7.1` (but it seems more accurate anyway).